app/sidekiq/lighthouse/create_intent_to_file_job.rb

```diff
@@ -51,14 +55,20 @@ class CreateIntentToFileError < StandardError; end
     # @param [ActiveSupport::TimeWithZone] form_start_date
     # @param [String] veteran's ICN
     #
-    def perform(form_type, form_start_date, veteran_icn)
-      init(form_type, veteran_icn)
+    def perform(in_progress_form_id, veteran_icn, participant_id)
+      init(in_progress_form_id, veteran_icn)
 
-      @itf_log_monitor.track_create_itf_begun(@itf_type, form_start_date, @user_account&.id)
+      @itf_log_monitor.track_create_itf_begun(@itf_type, @form_start_date, @user_account&.id)
       @service.create_intent_to_file(@itf_type, '')
-      @itf_log_monitor.track_create_itf_success(@itf_type, form_start_date, @user_account&.id)
+      @itf_log_monitor.track_create_itf_success(@itf_type, @form_start_date, @user_account&.id)
+    rescue MissingICNError, MissingParticipantIDError
```

Comment:
catch the other 2 custom errors as well? log a failure/exhaustion, but would not want to retry a job we know will continue to fail

Reply:
I agree i can also rescue the invalid ITF type error, but for user account not found, there's a chance that a user account for the given ICN can be created in between retries, resulting in a success, so i'll continue to re-raise that error. I'll remove the custom error and just have it raise a record not found error.